mojap_metadata/converters/sqlalchemy_converter/README.md

```diff
@@ -0,0 +1,120 @@
+# SQLAlchemy Converter
+
+[Functions](#functions)
+[Connection](#connection)
+[Data Types](#data-types)
+[Testing](#testing)
+
+Uses the SQLAlchemy [`Inspector`](https://docs.sqlalchemy.org/en/20/core/reflection.html#fine-grained-reflection-with-inspector) class to:
+
+1. Extract metadata from database dialects supported by [SQLAlchemy](https://docs.sqlalchemy.org/en/20/dialects/index.html#dialects)
+2. Convert the extracted output into a mojap `Metadata` object
+
+Currently assumes SQLAlchemy v1.4 but compatible with SQLAlchemy v2.0.
```

Comment:
might want to add a link to explain the difference in versions. Given that v1.4 is the legacy version, it may be better to say "compatible with SQLalchemy v1.4 and v2.0", unless you want to expand on what assumptions are involved when you say it "assumes" v1.4